lib/StringClient.js

```diff
@@ -89,6 +95,15 @@ StringClient.prototype.read = function read(options, callback) {
         }
 
         req.once('result', self.parse(req, callback));
+        req.once('redirect', function (res) {
+            res.resume();
```

Comment:
I did not call it. I have called .resume to consume data from res object. I found this on node docs:

> If no 'response' handler is added, then the response will be entirely discarded. However, if you add a 'response' event handler, then you must consume the data from the response object, either by calling response.read() whenever there is a 'readable' event, or by adding a 'data' handler, or by calling the .resume() method.

https://nodejs.org/dist/latest-v4.x/docs/api/http.html#http_class_http_clientrequest

Is this wrong?